### PR TITLE
libdmapsharing: update to 2.9.38

### PR DESCRIPTION
**Pull request by MikePetullo**
Maintainer: me
Compile tested: x86, x86_64, LEDE master
Run tested:  x86, x86_64, LEDE 17.01.1

Description:
libdmapsharing: update to 2.9.38

### CODE REVIEW

**Comment by diizzyy** [comment]
...and this is used by?

File: libs/libdmapsharing/Makefile [hunk 1]
```diff
@@ -20,7 +20,7 @@ PKG_LICENSE_FILES:=COPYING
 
 PKG_SOURCE:=libdmapsharing-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.flyn.org/projects/libdmapsharing/
```
Comment (diizzyy) [inline libs/libdmapsharing/Makefile]: Switch to HTTPS

File: libs/libdmapsharing/Makefile [hunk 2]
```diff
@@ -20,7 +20,7 @@ PKG_LICENSE_FILES:=COPYING
...
-PKG_MD5SUM:=02008e1998646d0bb6ca45784133b748
+PKG_MD5SUM:=2494161340c4c3c36907359eeddc4da3
```
Comment (diizzyy) [inline libs/libdmapsharing/Makefile]: Change to PKG_HASH (SHA256)

**Comment by hnyman** [comment]
Next time, please switch to using sha256 PKG_HASH like diizzyy suggests. 
PKG_MD5SUM is being phased out.

**Pull request merged**